mogoria/Politico
# #163740684 fix flask setup

#### What does this PR do?
fixes flask setup

#### Description of Task to be completed?
Remove bug that prevents flask from running
Refactor code to include pep8 style guide

#### How should this be manually tested?

- Clone this repository
- run the command `export FLASK_APP=run.py`
- run the command `flask run` in the terminal
- using postman, send a get request to `http://127.0.0.1:5000/api/v1`
- you should see a JSON response with the message, welcome

#### What are the relevant pivotal tracker stories?
[#163740684](https://www.pivotaltracker.com/n/projects/2241724/stories/163740684)
#### Screenshots (if appropriate)
#### Questions:
